docgen/src/guides/migration.md

```diff
@@ -88,6 +88,16 @@ As of now, we consider the engine used to build the widgets in InstantSearch.js
 as an implementation detail. Since we do not expose it anymore, we'll be able
 to change it and use the best solution for each release.
 
+## RangeSlider widget is using Rheostat as Slider Component
+
+Since we had unclosed issues with the old slider, we switched to a new
```

Comment:
I don’t agree, we can say that it’s better for the reasons that it’s better, but I don’t think we should say that it was bad before.